packages/datadog-instrumentations/src/ws.js

```diff
@@ -0,0 +1,136 @@
+'use strict'
+
+const {
+  addHook,
+  channel
+} = require('./helpers/instrument')
+const shimmer = require('../../datadog-shimmer')
+
+const tracingChannel = require('dc-polyfill').tracingChannel
+const serverCh = tracingChannel('ws:server:connect')
+const producerCh = tracingChannel('ws:send')
+const receiverCh = tracingChannel('ws:receive')
+const closeCh = tracingChannel('ws:close')
+const emitCh = channel('tracing:ws:server:connect:emit')
+
+function wrapHandleUpgrade (handleUpgrade) {
+  return function () {
+    const [req, socket, , cb] = arguments
+    if (!serverCh.start.hasSubscribers || typeof cb !== 'function') {
+      return handleUpgrade.apply(this, arguments)
+    }
+
+    const ctx = { req, socket }
+
+    arguments[3] = function () {
+      return serverCh.asyncStart.runStores(ctx, () => {
+        try {
+          return cb.apply(this, arguments)
+        } finally {
+          serverCh.asyncEnd.publish(ctx)
+        }
+      }, this, ...arguments)
+    }
+    return serverCh.traceSync(handleUpgrade, ctx, this, ...arguments)
+  }
+}
+
+function wrapSend (send) {
+  return function wrappedSend (...args) {
```

Comment:
```suggestion
  return function wrappedSend () {
```

I don't think `args` is being used and should be safe to remove. That said I'm not sure why the linter didn't complain.